app/models/katello/authorization/sync_plan.rb

```diff
@@ -16,10 +16,26 @@ def deletable?
       authorized?(:destroy_sync_plans)
     end
 
+    def syncable?
+      products.each { |p| p.syncable? }
```

Comment:
does this make sense ? would you want this to return `[true, false]` ?

Reply:
Updated..just need a true or false..